docs/grafana.rst

```diff
@@ -0,0 +1,320 @@
+Real time graphs
+====================
+
+Grafana
+-----------
+
+`Grafana <https://grafana.com/>`_ is the open source analytics and monitoring solution and may be used to view various BEAM metrics in real time during BEAM execution.
+
+Grafana is a tool to display data from data source. In our case the data source is `InfluxDB <https://www.influxdata.com/products/influxdb-overview/>`_ .
+
+
+Docker installation and configuration
+--------------------------------------
+
+There is a docker image with configured Grafana and InfluxDB to work with BEAM thus one need to install docker in order to use Grafana.
+
+Ubuntu Linux users
+^^^^^^^^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* 64-bit version of one of these Ubuntu versions: Disco 19.04, Cosmic 18.10, Bionic 18.04 (LTS), Xenial 16.04 (LTS)
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Ubuntu Linux users <https://docs.docker.com/install/linux/docker-ce/ubuntu/>`_ and
```

Comment:
Why is this across multiple lines?

Reply:
For better readability during documentation writing only.
In any case it is parsed correctly:

![image](https://user-images.githubusercontent.com/5393513/78022517-bdb30400-735d-11ea-8676-62f6fb88b544.png)

Reply:
this is how it looks in editor:

![image](https://user-images.githubusercontent.com/5393513/78033930-bdbbff80-736f-11ea-8f6d-8c8c57e0e6d4.png)

Reply:
I guess I'm just not sure how it looks as window size increases and decreases. Artificial splits when shrunk can look weird